.gitignore

```diff
@@ -8,7 +8,7 @@ yarn.lock
 package-lock.json
 
 # Misc
-sandbox
+example
```

Comment:
We're not using either of these are we?

Reply:
@okwolf I am locally, but it's not committed to the repo.